### DISCUSSION TITLE
Reading the Queue Sizes of Redis

### DISCUSSION THREAD

**Discussion by mast3rbow** [django-q2/django-q2#135] [Q&A]
Hi, i've been running with a Redis broker, however it seems to always report Queue size of 0, and the cluster shows a Queue size of 0 and a done queue size of 0.

I was wondering if anyone else has noticed this same behaviour. I remember reading somewhere in the old documentation about limitations of reading queues with redis cache's but can't locate that information again.

**Comment by GDay** [comment]
Are you using MacOS? It will likely return 0 there:

> Uses [multiprocessing.Queue.qsize()](https://docs.python.org/3.8/library/multiprocessing.html#multiprocessing.Queue.qsize) which is not implemented on OS X and always returns 0.

Footnote on this page: https://django-q2.readthedocs.io/en/master/monitor.html

**Comment by mast3rbow** [comment]
Hi Stan,

This is actually on Azure Linux Webapp, with a redis cache. I'm also not able to get it my dev box which is Windows.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
